src/lib/preferences-default-codes.ts

```diff
@@ -27,7 +29,7 @@ const ywLogo = `
 // ---------------------------------------------------------------------------------------------
 // ----------------- Common javascript/typescript of preferences -------------------------------
 // ---------------------------------------------------------------------------------------------
-
+export const isVisitor$ = new BehaviorSubject(true)
```

Comment:
If the 'visitor$' concept is to be included in os-core, it should be fully implemented, e.g.:
```
// isVisitor$ is an observable that emit wether or not the current user is a visitor
export const isVisitor$ = new AssetsGateway.Client().accounts
    .getSessionDetails$()
    .pipe(
        raiseHTTPErrors(),
        map((v) => v.userInfo.temp,
    )
```
Then, another question on where to expose it in the package should be answered (not sure that exposing it at the root of the package is the right choice, not sure where, but it can be discussed).

For now, I would just keep the 'visitor$' concept out of os-core - for now there is only '@youwol/platform' that needs it - and reconsider it latter if this need comes over and over again in other packages.